README.md

```diff
@@ -1,75 +1,111 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
 
+# Microverse
+### Prerequisites
 
-## Built With
+- Basic html, css and medium Javascript know how.
+- Git installation
+- A text editor 
+- A terminal
+- A web browser to view output e.g Google Chrome
+- An IDE e.g Visual studio code
 
-- Major languages
-- Frameworks
-- Technologies used
+#### I have implement the following interactions:
 
-## Live Demo (if available)
+Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
+Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
+Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
 
-[Live Demo Link](https://livedemo.com)
 
+<img alt="image" src="https://user-images.githubusercontent.com/88346506/191383778-fe90978d-d70d-4c83-85e5-ca4ff3364d08.png">  <img alt="image" src="https://user-images.githubusercontent.com/88346506/191384000-b76aa8d0-570f-4c9d-a2cd-e822b5497784.png">
 
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## Description
+For this repository, I've created html templates and styling files in order to built a responsive Microverse capstone.
 
+## Built With
+- `HTML`
+- `CSS`
+- `JS`
+- `Git & Github`
+<img alt="image" src="https://user-images.githubusercontent.com/88346506/191978106-8371549b-99c6-48c5-856a-1657998313cb.png">   <img alt="image" src="https://user-images.githubusercontent.com/88346506/191979506-46d7a9f5-db8b-4f21-a3c0-d8b580e0469f.png">
 
+## Getting Started
 To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+- Create a new GitHub repo.
+- Create a new branch according to the GitHub flow rules.
+- Add a .gitignore file.
+- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+- Add descriptive README file to your project - please use this template but remember to customize it to your project.
+A .css file that includes one class with styles for your header .
+- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+- Open a pull request.
```

Comment:
[Optional] I recommend that you draft more appropriate setup instructions. Setup instructions guide the public on how to run your program locally on their machines. You seem to have copied the project development instructions which is different from setup instructions.

Reply:
thanks but this did not stop approving the project